Modules/Menu/Tests/MenuItemUriGeneratorTest.php

```diff
@@ -43,7 +43,7 @@ public function it_generates_basic_uri_without_parent()
     /** @test */
     public function it_generates_uri_with_the_parents_slug()
     {
-        Event::fake();
+        //Event::fake();
```

Comment:
Could you remove the comment altogether, please :)

Reply:
done!